doc/tutorials/content/building_pcl.rst

```diff
@@ -176,7 +176,6 @@ The available ROOTs you can set are as follow:
 * **CMINPACK_ROOT**: for cminpack with value `C:/Program Files/CMINPACK 1.1.13` for instance
 * **QHULL_ROOT**: for qhull with value `C:/Program Files/qhull 6.2.0.1373` for instance
 * **FLANN_ROOT**: for flann with value `C:/Program Files/flann 1.6.8` for instance
-* **EIGEN_ROOT**: for eigen with value `C:/Program Files/Eigen 3.0.0` for instance
```

Comment:
Setting `EIGEN_ROOT` wouldn't work anymore. Whoever installes Eigen to an exotic location would need to set `Eigen3_DIR` properly as listed below.